arches_querysets/rest_framework/serializers.py

```diff
@@ -428,7 +438,9 @@ def validate(self, attrs):
 class ArchesTileSerializer(serializers.ModelSerializer, NodeFetcherMixin):
     # These fields are declared here in full instead of massaged via
     # "extra_kwargs" in class Meta to support subclassing by TileAliasedDataSerializer.
-    tileid = serializers.UUIDField(validators=[], required=False, allow_null=True)
+    tileid = serializers.UUIDField(
+        validators=[], required=False, initial=uuid.uuid4, allow_null=True
```

Comment:
We can revert this change if we don't want tileid's generated on the blanks, but they were being generated for resources, so it was asymmetrical.

Reply:
Discussed in modular reports planning 2025-08-04, we don't want tileid's generated on the blanks, will revert.

Reply:
503efd9aefa6dbb00f3cd09b546415caa5a76544